manager/notifier.go

```diff
@@ -190,6 +191,15 @@ func (n *notifier) sendEmailNotification(email string, a *Alert) error {
 	return writeEmailBody(wc, a)
 }
 
+func (n *notifier) sendPushoverNotification(token, userKey string, a *Alert) error {
+	po, err := pushover.NewPushover(token, userKey)
+	if err != nil {
+		return err
+	}
+
+	return po.Message("Alertmanager " + a.Description)
```

Comment:
Hmm, looking at https://pushover.net/api#messages, pushover seems to support titles for messages, which would be great to put the alert Summary in. But I guess the problem is that the client library so far doesn't support setting a title?